src/dosext/dpmi/memory.c

```diff
@@ -60,11 +60,11 @@ static int in_dpmi_pool(dosaddr_t base, unsigned int size)
     return 0;
 }
 
-void dpmi_set_mem_base(void *base)
+void dpmi_set_mem_base(void)
 {
     dpmi_lin_rsv_base = MEM_BASE32(LOWMEM_SIZE + HMASIZE);
-    dpmi_base = base;
-    c_printf("DPMI memory mapped to %p\n", base);
+    dpmi_base = MEM_BASE32(config.dpmi_base);
```

Comment:
Then probably this function can be
eliminated, and its content moved to
dpmi_alloc_pool() or dpmi_setup()
or alike?